pkg/spi-shared/config/config.go

```diff
@@ -75,29 +112,50 @@ type SharedConfiguration struct {
 // mainly contains config.yaml of the OAuth application within the service provider.
 type ServiceProviderConfiguration struct {
 	// ClientId is the client ID of the OAuth application that the SPI uses to access the service provider.
-	ClientId string `yaml:"clientId"`
+	ClientId string
 
 	// ClientSecret is the client secret of the OAuth application that the SPI uses to access the service provider.
-	ClientSecret string `yaml:"clientSecret"`
+	ClientSecret string
 
 	// ServiceProviderType is the type of the service provider. This must be one of the supported values: GitHub, Quay
-	ServiceProviderType ServiceProviderType `yaml:"type"`
+	ServiceProviderType ServiceProviderType
 
 	// ServiceProviderBaseUrl is the base URL of the service provider. This can be omitted for certain service provider
 	// types, like GitHub that only can have 1 well-known base URL.
-	ServiceProviderBaseUrl string `yaml:"baseUrl,omitempty"`
+	ServiceProviderBaseUrl string
 
 	// Extra is the extra configuration required for some service providers to be able to uniquely identify them. E.g.
 	// for Quay, we require to know the organization for which the OAuth application is defined for.
```

Comment:
Could you please remove this incorrect comment about quay?

Reply:
fixed